lib/bootloader_setup.pm

```diff
@@ -581,6 +583,21 @@ sub bootmenu_default_params {
         }
         push @params, "Y2DEBUG=1";
     }
+    elsif (get_var('AGAMA')) {
+        wait_screen_change { send_key "e" };
+        send_key "down";
+        send_key "down";
+        send_key "down";
+        send_key "down";
```

Comment:
A minor comment here, could you use `send_key_until_needlematch` instead? anyway, LGTM

Reply:
@rfan1 I'll make it since @Martchus mentioned the same :-) And if two people already see the same problem, then it seems to be a legit one.

Reply:
@rfan1 but we're talking about needling until the _ is in front of linux entry ... I'm really afraid that this will cause more future work with re-needling especially if font/theme changes. While the send_key down 4 times, will work probably forever unless we reoder grub which usually doesn't happen in lifespan of a major version (e.g. 16.X or 15.X).

![image](https://github.com/user-attachments/assets/2a5a7baa-1db4-4b2e-86f8-94288c83760b)